Presentation.Web/Models/API/V1/Organizations/ChangeOrganizationRegistrationRequestDTO.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+
+namespace Presentation.Web.Models.API.V1.Organizations
+{
+    public class ChangeOrganizationRegistrationRequestDTO
```

Comment:
for all of these "*OrganizationRegistrationRe**" DTO's... should it not be "Organization**Unit**Registrations...."
check all of your dtos